File: SoloLearn/Jungle Camping/JungleCamping.rb
sounds = gets.chomp.split
animals = ""
for sound in sounds
    case sound
    when "Grr"
        animals += "Lion "
    case "Rawr"
        animals += "Tiger "
    case "Ssss"
        animals += "Snake "
    case "Chirp"
        animals += "Bird "
    end
end
puts animals[0..-2]

File: SoloLearn/Jungle Camping/JungleCamping.cpp
#include <iostream>
using namespace std;

int main() {
    string sound, animals;
    while (cin >> sound) {
        if (sound == "Grr")
            animals += "Lion ";
        else if (sound == "Rawr")
            animals += "Tiger ";
        else if (sound == "Ssss")
            animals += "Snake ";
        else if (sound == "Chirp")
            animals += "Bird ";
    }
    animals.pop_back();
    cout << animals;
    return 0;
}
